File: Modul_2/Soal_4/PRAK204-2310817210015-MuhammadBukhariFitri.py
userInput = input().split()

# Test Case 1
if len(userInput) == 1:
    jariJari = float(userInput[0])
    tinggi = float(input())

# Test Case 2
elif len(userInput) == 2:
    jariJari, tinggi = map(float, userInput)

phi = 22 / 7
volume = phi * jariJari * jariJari * tinggi
luas = 2 * phi * jariJari * (jariJari + tinggi)
keliling = 2 * phi * jariJari

print(f"Volume = {volume:.2f}")
print(f"Luas = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")

File: Modul_2/Soal_4/PRAK204-2310817210015-MuhammadBukhariFitri.c
#include <stdio.h>

int main() {
    float jariJari;
    float tinggi;
    float phi = 22.0 / 7.0;
    scanf("%f", &jariJari);
    scanf("%f", &tinggi);

    float volume = phi * jariJari * jariJari * tinggi;
    float luas = 2 * phi * jariJari * (jariJari + tinggi);
    float keliling = 2 * phi * jariJari;

    printf("Volume = %.2f\n", volume);
    printf("Luas = %.2f\n", luas);
    printf("Keliling = %.2f", keliling);
}